samples/tab-product-inspection/csharp/README.md

```diff
@@ -0,0 +1,50 @@
+﻿# Product Inspection
+
+This sample app demonstrate a feature where user can scan a product and mark it as approved/rejected.
```

Comment:
This sample app demonstrate a feature where user can scan a product, capture a image and mark it as approved/rejected.